mod/groups/views/default/resources/groups/requests.php

```diff
@@ -27,11 +22,17 @@
 	'entity' => $group,
 ]);
 
-$params = [
-	'content' => $content,
+$tabs = elgg_view_menu('groups_members', [
+	'entity' => $group,
+	'class' => 'elgg-tabs'
+]);
+
+// build page
+$body = elgg_view_layout('content', [
```

Comment:
default

Reply:
was already a `content` layout, not changing it now